hydra-node/test/Hydra/FireForgetSpec.hs

```diff
@@ -4,27 +4,25 @@ import Cardano.Prelude hiding (atomically, concurrently)
 
 import Control.Monad.Class.MonadAsync (concurrently)
 import Control.Monad.Class.MonadSTM (MonadSTM (..), TVar, readTVar, writeTVar)
-import Control.Monad.IOSim (IOSim, runSimOrThrow)
 import Control.Tracer (nullTracer)
-import Network.TypedProtocol.Channel (createConnectedChannels)
-import Network.TypedProtocol.Driver.Simple (runPeer)
 import Hydra.Network.Ouroboros.Client (FireForgetClient (..), fireForgetClientPeer)
 import Hydra.Network.Ouroboros.Server (FireForgetServer (..), fireForgetServerPeer)
 import Hydra.Network.Ouroboros.Type (codecFireForget)
-import Test.Hspec
+import Network.TypedProtocol.Channel (createConnectedChannels)
+import Network.TypedProtocol.Driver.Simple (runPeer)
+import Test.Hspec.Core.Spec
+import Test.Util (shouldBe, shouldRunInSim)
```

Comment:
Why not using `Test.Hspec` which is the more general public interface of Hspec?

Reply:
Perhaps we could reexport `Test.Hspec` from `Test.Util` in order to start building a "Test Prelude" unifying how we write and run tests in Hydra?

Reply:
This is a relict from where I tried to shuffle things a bit.. I like the idea of a `Test.Prelude` though!